lib/utils/qemu.ts

```diff
@@ -175,30 +188,27 @@ export async function installQemuIfNeeded(
  * - https://stackoverflow.com/questions/55388725/run-linux-arm-container-via-qemu-binfmt-misc-on-docker-lcow
  *
  * @param docker Dockerode instance
+ * @param emulated The --emulated command-line option
+ * @param logger Logger instance
```

Comment:
Should we delete the jsdoc params lists?

Reply:
Why, because they are not useful? The `docker` and `logger` descriptions are not useful indeed, but the `emulated` description actually clarifies that it is the `--emulated` command line option.

Reply:
I thought I remember you suggesting we stop adding them, since we are using TS.  I don't mind though.

Reply:
Related to this, what I remember is that some param tags inherited from typed javascript files (.js) include the type hinting that becomes redundant when converted to typescript. Example:

`@param {boolean} emulated The --emulated command-line option`
_vs._
`@param emulated The --emulated command-line option`

So the type hinting should be deleted when converted to Typescript, especially as it introduces the possibility of conflict with the Typescript type specification. But the parameter descriptions may be useful, if they add some value ("Logger instance" does not add value, granted!)